core/ir/aarch64/instr_create_api.h

```diff
@@ -463,12 +463,12 @@
  * they just need to know whether they need to preserve the app's flags, so maybe
```

Comment:
Please enable the samples that use this which are currently disabled:
```
  # dr_insert_cbr_instrument_ex is NYI
  add_sample_client(cbrtrace    "cbrtrace.c;utils.c"    "drmgr;drx")
  add_sample_client(hot_bbcount "hot_bbcount.c"         "drmgr;drreg;drbbdup;drx")
```

Those are run as tests, though w/o targeted correctness checks: just making sure they don't crash.